calico-vpp-agent/cni/storage/storage.go

```diff
@@ -238,8 +244,29 @@ func (hp *HostPortBinding) String() string {
 	return s
 }
 
+/* 8 base64 character hash */
+func hash(text string) string {
+	h := sha512.Sum512([]byte(text))
+	return base64.StdEncoding.EncodeToString(h[:])[:vrfTagHashLen]
+}
+
+func truncateStr(text string, size int) string {
+	if len(text) > size {
+		return text[:size]
+	}
+	return text
+}
+
+func (ps *LocalPodSpec) GetVrfTag(ipFamily vpplink.IpFamily) string {
+	h := hash(fmt.Sprintf("%s%s", ps.NetnsName, ipFamily.ShortStr))
```

Comment:
Why don't we include the interface name in the hashed string / tag for the VRF? Won't we have identical tags if a pod requests 2 interfaces (multi-net or just plain multus) ?

Reply:
Right, fixed :)